src/systems/analysis_points.jl

```diff
@@ -0,0 +1,645 @@
+"""
+    $(TYPEDEF)
+    $(TYPEDSIGNATURES)
+
+Create an AnalysisPoint for linear analysis. Analysis points can be created by calling
```

Comment:
```suggestion
Create an AnalysisPoint for input-output and linear analysis. Analysis points can be created by calling
```

I'd like to widen the scope to handle any IO analysis